cypress/integration/code-schools.spec.js

```diff
@@ -2,16 +2,39 @@ describe('code schools', function() {
   beforeEach(() => {
     cy.server();
     cy.route('GET', '/api/v1/code_schools').as('codeSchools');
-
     cy.visitAndWaitFor('/code_schools');
-
     cy.get('h1').should('have.text', 'Code Schools');
+    cy.wait('@codeSchools');
   });
 
   it('should render many code school cards', () => {
-    cy.wait('@codeSchools');
-
     // 40 is arbitrary, but it proves that the API is working and leading to rendered content
     cy.get('[data-testid="SchoolCard"]').should('have.length.greaterThan', 40);
   });
+
+  it('should render Cincy and Tech Elevator after selecting Ohio', () => {
+    cy.get('input#react-select-state_select-input')
+      .type('Ohio', { force: true })
+      .type('{enter}');
+
+    cy.get('[data-testid="SchoolCard"]')
+      .find('img[alt="Cincy Code IT Bootcamps logo"]', 'img[alt="Tech Elevator logo"]')
```

Comment:
You can be even simpler:

```javascript
cy.get('[data-testid="SchoolCard"]')
    .find('Cincy Code IT Bootcamps', 'Tech Elevator')
    .should('exist');
```

Reply:
Ah ok. .find for the text.

Reply:
completed

Reply:
This threw an error, so I had to use
```JS  
.should('contain', 'Cincy Code IT Bootcamps')
```
which checked to make sure the SchoolCard contained the string.